docs/src/user-guide/guides-using-object-storage/clp-usage.md

```diff
@@ -11,22 +11,21 @@ angle brackets (`<>`) with the appropriate values:
 ```bash
 sbin/compress.sh \
   --timestamp-key <timestamp-key> \
-  <prefix>
+  https://<bucket-name>.s3.<region-code>.amazonaws.com/<prefix>
```

Comment:
thinking if we can improve this doc. I will let you know if I come up with something better.